src/base/stringUtils.cpp

```diff
@@ -147,7 +147,7 @@ doublereal fpValueCheck(const std::string& val)
     ch = str[0];
     if (ch == '+' || ch == '-') {
         if (str.size() == 1) {
-            throw CanteraError("fpValueCheck", "string ends in '{}'", ch);
+            throw CanteraError("fpValueCheck", "string '{}' ends in '{}'", val, ch);
```

Comment:
Would it make sense to start using f-strings in those instances?

Reply:
What? This is in C++...

Reply:
oops! nevermind ... apparently coded too much Python lately. 

In that case, this all looks fine to me!!